source/components/organisms/SummaryList/SummaryList.tsx

```diff
@@ -40,6 +40,16 @@ const SumContainer = styled.View<{ colorSchema: string }>`
   background: ${(props) =>
     props.theme.colors.complementary[props.colorSchema][3]};
 `;
+
+const SummarySpacer = styled.View<{ colorSchema: string }>`
+  margin-bottom: 16px;
+  height: 4px;
+  width: 32px;
+
+  background: ${(props) =>
+    props.theme.label.colors[props.colorSchema].underline};
+`;
+
```

Comment:
Will this give you any errors?

```js
import {
  PrimaryColor,
  ThemeType,
} from "../../../styles/themeHelpers";

....

const SummarySpacer = styled.View<{
  colorSchema: PrimaryColor;
  theme: ThemeType;
}>`
  margin-bottom: 16px;
  height: 4px;
  width: 32px;

  background: ${(props) =>
    props.theme.label.colors[props.colorSchema].underline};
`;
```

Reply:
Not an error per se, but syntax highlighting and intellisense completely stops working

![image](https://user-images.githubusercontent.com/10225982/172318861-110f33a2-c9d5-42ed-910a-dd4a4c184ea1.png)

Reply:
Fixed by adding an interface instead of inline typings